apps/hash-ai-worker-py/turbo.json

```diff
@@ -6,24 +6,19 @@
       "persistent": true
     },
     "poetry:venv": {
-      "outputs": ["./.venv/**"],
-      "inputs": ["pyproject.toml", "poetry.lock"]
+      "cache": false
```

Comment:
I understand wanting to avoid debugging this issue, but I think this will cause a reinstall on every single Python turbo command (such as `black`, `ruff`, etc.)

Do we perhaps want to remove the `dependsOn` from those, or remove the `cache` here and rely on it checking the whole workspace for cache invalidation? etc.

Reply:
Removing the `dependsOn` might be sensible, yes, but I don't understand what you mean by checking the whole workspace for cache invalidation. Do you mean add `pyproject.toml` and `poetry.lock` to the global invalidation logic? This _could_ work as well.

Reply:
This does not work reliably, I removed the dependencies on the install script. For `yarn` we have the same behavior: we have to call `yarn install`.